Dangerfile

```diff
@@ -61,3 +61,20 @@ rescue => ex
   fail "There was an error with Danger bot's Junit parsing: #{ex.message}"
   message ex.inspect # view the entire error output in the log
 end
+
+# Store screenshots in Google Cloud
+require "google/cloud/storage"
+storage = Google::Cloud::Storage.new project_id: "public-lab"
+bucket  = storage.bucket "plots2-screenshots"
+Encoding.default_external = 'UTF-8'
```

Comment:
This may need: `# encoding: utf-8` as per https://stackoverflow.com/questions/20521371/set-utf-8-as-default-for-ruby-1-9-3#20523482

Reply:
`Encoding.default_internal = Encoding::UTF_8`

Reply:
>     Ruby 2.0 and above: default string encoding is UTF-8.

> Thus, if you use Ruby 2.0, you could skip the encoding comment and correctly assume UTF-8 encoding everywhere by default.

Reply:
Yes maybe as a follow-up?